### PR TITLE
Bel-4070-1 Change Running Task Alarm to use a threshold of 100

### PR DESCRIPTION
**Pull request by JoshuaGleason**
[Link to Jira ticket](https://strongmind.atlassian.net/browse/Bel-4070)

## Purpose 

Adjust alarm to use 100 as the threshold
## Approach 

Change the threshold to 100
## Testing

Tests pass
Deployed locally on repo-dashboard
## Screenshots/Video

image

### CODE REVIEW

File: deployment/src/strongmind_deployment/container.py [hunk 1]
```diff
@@ -69,6 +69,7 @@ def __init__(self, name, opts=None, **kwargs):
         self.max_capacity = 100
         self.min_capacity = self.desired_count
         self.sns_topic_arn = kwargs.get('sns_topic_arn')
+        self.binary_sns_topic_arn = 'arn:aws:sns:us-west-2:221871915463:DevOps-Opsgenie'
```
Comment (tpaulshippy) [inline deployment/src/strongmind_deployment/container.py:72]: Is this change intentional?

Comment (JoshuaGleason) [reply deployment/src/strongmind_deployment/container.py:72]: Yes

**Review by tpaulshippy — APPROVED** [review]
